src/query/management/src/procedure/procedure_mgr.rs

```diff
@@ -64,12 +70,14 @@ impl ProcedureMgr {
         let meta = &req.meta;
         let name_ident_raw = serialize_struct(name_ident.procedure_name())?;
 
+        let mut old_id = None;
         let create_res = self
             .kv_api
             .create_id_value(
                 name_ident,
                 meta,
                 overriding,
+                &mut old_id,
```

Comment:
Better refactor create_id_value and delete the ownership in overriding logic. Now I just return a old_id.